ex1.py

```diff
@@ -0,0 +1,9 @@
+#area of a circle
+import math
+def Area(r):
```

Comment:
function names start with NONE capital letters, "Area" should be changed to "area"